test/parallel/test-debug-args.js

```diff
@@ -2,6 +2,6 @@
 // Flags: --debug-code
 
 require('../common');
-var assert = require('assert');
+const assert = require('assert');
 
-assert.notEqual(process.execArgv.indexOf('--debug-code'), -1);
+assert(!process.execArgv.includes('--debug-code'));
```

Comment:
Well, that's certainly embarrassing. I must have tried to do a shortcut by only running the test I changed, and then pasted in the wrong file name or something. Fixing momentarily.